SU-SWS/react_paragraphs
# D8CORE-1443 Fixed links with arbitrary text

# READY FOR REVIEW

# Summary
- Fixed link widgets when the user puts in random strings
- Updated underlying depdendencies
- Added a row to the bottom of the list if the last row is full. this reduces one step for the user

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. Checkout this branch
1. add a card to a basic page
1. enter some text into the link field and save after each input to verify no errors.
   - Try actual urls `http://google.com` and `/about`
   - Try autocompleted links: type `Abo` and choose the link to the about page
   - Try random text `This is garbage, but still works`
 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
